student/src/main/java/tds/student/sql/repository/RemoteExamRepository.java

```diff
@@ -98,6 +99,15 @@ private NoContentResponseResource handleErrorResponseNoContent(String body) thro
     }
   }
   
+  private Response<ExamConfiguration> handleErrorResponseExamConfig(String body) throws ReturnStatusException {
+    try {
+      JavaType type = objectMapper.getTypeFactory().constructParametricType(Response.class, Exam.class);
+      return objectMapper.readValue(body, type);
+    } catch (IOException e) {
+      throw new ReturnStatusException(e);
```

Comment:
Could/should we include the body as a property in the ReturnStatusException?  We were running into debugging issues on my last project, where we weren't able to deserialize the payload but we didn't know why because we couldn't see the payload.